app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,46 @@
+/* Space out content a bit */
+body {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+/* Everything but the jumbotron gets side spacing for mobile first views */
+.header {
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+/* Custom page header */
+.header {
+  border-bottom: 1px solid #e5e5e5;

```

Comment:
Color literals like `#e5e5e5` should only be used in variable declarations; they should be referred to via variable everywhere else.